src/reducers/polling.js

```diff
@@ -215,13 +215,11 @@ export const updateVoteBreakdown = pollId => (dispatch, getState) => {
 export const getVoteBreakdown = async (pollId, options, endDate) => {
   // returns either the block on which this poll ended,
   // or, if the poll hasn't ended, the current block
-  const pollEndBlock = await window.maker
-    .service('govQueryApi')
-    .getBlockNumber(Math.floor(endDate.getTime() / 1000));
+  const pollEndUnix = Math.floor(endDate / 1000);
```

Comment:
what do you think about keeping the `endDate. getTime()` here? just to be slightly more explicit about what's happening to the `endDate` date obj